util/src/main/java/net/automatalib/util/partitionrefinement/PaigeTarjan.java

```diff
@@ -282,14 +282,18 @@ public void computeCoarsestStablePartition() {
         return b;
     }
 
+    /**
+     * Move the state to the left of its Block ptr. This allows grouping of states with similar behavior.
+     * @param state - state to be moved left within its Block.
```

Comment:
I think the - will result in some weird markup in the generated JavaDoc. I'd prefer to just format it like the rest of the code. So something like:
```java
* @param state
*         state to be moved left within its Block
```